tensorflow/lite/micro/kernels/activations_test.cc

```diff
@@ -135,6 +137,312 @@ void TestRelu6Float(const int* input_dims_data, const float* input_data,
   }
 }
 
+void GenerateUniformRandomVector(int size, float min, float max,
+                                 std::minstd_rand* random_engine,
+                                 std::vector<float>* result) {
```

Comment:
std::vector uses malloc under the hood.  Avoid on TF Micro.  Follow the example in other tests by passing in static size input / output arrays.

Reply:
Done